.github/workflows/run-scrapers.yml

```diff
@@ -0,0 +1,127 @@
+name: Run scrapers
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 0 * * *'
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - id: import_gpg
+      uses: crazy-max/ghaction-import-gpg@v5
+      with:
+        gpg_private_key: |
```

Comment:
shouldn't we use keyvault for this ? ask Ben for access

Reply:
KeyVault is secrets for Azure. As this is using GitHub Actions, we need to use GitHub Secrets.